src/features/shields/media/arrowDown.tsx

```diff
@@ -0,0 +1,30 @@
+// THIS FILE IS AUTOGENERATED, DO NOT MODIFY MANUALLY
+/* tslint:disable */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License. v. 2.0. If a copy of the MPL was not distributed with this file.
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import GenerateComponentForGraphic from './template'
+
+// Direct access to SVG
+export const Graphic = (
+  <svg width='24' height='24' xmlns='http://www.w3.org/2000/svg' xmlnsXlink='http://www.w3.org/1999/xlink'>
+    <defs>
+      <path d='M12 13.262l4.762-4.081a.75.75 0 1 1 .976 1.138l-5.25 4.5a.75.75 0 0 1-.976 0l-5.25-4.5a.75.75 0 1 1 .976-1.138L12 13.262z'
+      id='a' />
```

Comment:
Can we try to avoid ID attributes on inline elements, poluting global namespace. This will cause problems if there's multiple SVG files on the same page trying to do something with '#a' and '#b'. I beleive this is an issue from the sketch export, and it could either be fixed in sketch, or manually (which I've had to do before 😠  )

Reply:
Also is this arrow different than our component arrow - if not, what's stopping you using the arrow component @cezaraugusto? cc @rossmoody